cypress/integration/pages/topicPage/tests.js

```diff
@@ -121,6 +135,30 @@ export default ({ service, pageType, variant }) => {
               });
           });
       });
+      it('clicking the message banner should navigate to the correct page', () => {
+        if (messageBanner) {
+          cy.go('back');
```

Comment:
Do we still need this?

Reply:
AFAIK this isn't covered under another test so I think so?

Reply:
Oh I meant whether the `cy.go('back')` command was still required - but I am assuming it must be?

Reply:
It was added because the test would have failed if not but perhaps it could be done in a diferent way. This could be refactored at a later date.